testsuite/run_sets/init_clients.yml

```diff
@@ -6,12 +6,8 @@
 
 ## Core - Initialize clients BEGIN ##
 
-# Proxy
-  # one of: proxy_register_as_minion_with_script.feature
-  #         proxy_register_as_minion_with_gui.feature
-  #         proxy_register_as_pod.feature
-- features/init_clients/proxy_register_as_minion_with_script.feature
-- features/init_clients/proxy_branch_network.feature
+- features/init_clients/proxy.feature
+#- features/init_clients/proxy_branch_network.feature
```

Comment:
It would be good to have a small comment above the commented out line that states why it is commented out.

Reply:
I also wonder, why do we have the proxy in 2 run sets?
- `init_clients.yml` vs.
- `min_proxy.yml`